### ISSUE TITLE
[1.4.2(31)] Xiaomi Redmi Note 3/5.1.1 User can not select images/videos from different folder.

### ISSUE THREAD

**Issue by Dipamit** [Lomotif/lomotif-android-issues#310]
Frequency: Every time
1. Open application and tap on Start.
2. Tap on Camera and select two images.
3. Tap on Back (Page navigate to Home screen)
4. Again tap on start and Select Whatsapp images.
5. Tap on Next and without selecting song again tap Next.
6. Verify preview screen.

Expected result: On tapping back page should not navigate to home screen. User should select all images and videos.

**Comment by vysohanych** [comment]
@xiuie I can't reproduce it on build 1.4.2 (31) on LG Nexus 5X Android 6.0.1.

**Comment by hiteshbheda** [comment]
@xiuie I can reproduce this issue for build [1.4.2 (31)] on Asus Zenfone Go / Android 5.1

**But I think it works as design and works perfectly. 
As when we click on "Back" button, it navigate user to Home screen and all selection will be cancelled. Again when you starts, it will be a new transaction (previous selection is not selected).**

**Comment by xiuie** [comment]
@Dipamit @hiteshbheda  I don't understand this bug. Are you saying you can't select images/video clips from multiple folders?

**Comment by testershub** [comment]
@xiuie Here the issue is user can not select images/videos from multiple folders as there is only one way to navigate back to "Add Photos & Videos" screen, i.e., by tapping on device back button (hard key). 

Expected result: User should be navigated back to "Add Photos & Videos" screen after tapping on the back icon (top left corner) of the application. 

Actual result: User is navigated to the home screen after tapping on back icon (top left corner) of the application by deselecting/discarding all the selected images/videos.

**Comment by hiteshbheda** [comment]
@xiuie Yes, I agree with @testershub . If we want to use images / videos from more than one folder then it's not possible from this flow.

**Comment by hiteshbheda** [comment]
@xiuie I can reproduce this issue for build [1.4.4 (33) Live Build] on Asus Zenfone Go 4.5 / Android 5.1

**Comment by hiteshbheda** [comment]
@xiuie I can reproduce this issue for build 1.4.4 (33) on Asus Zenfone Go / Android 5.1
